contracts/DaiHrd.sol

```diff
@@ -430,10 +423,10 @@ contract ERC777 is RuntimeConstants, Context, IERC777, IERC20 {
 }
 
 contract MakerFunctions {
-	uint constant ONE = 10 ** 27;
 	// KEYDONIX: Renamed from `rmul` for clarity
+	// KEYDONIX: Changed ONE to 10**27 for clarity
 	function safeMul27(uint x, uint y) internal pure returns (uint z) {
-		z = safeMul(x, y) / ONE;
+		z = safeMul(x, y) / 10 ** 27;
```

Comment:
I feel like `ONE` in a contract that uses 10**18, 10**27, and 10**45 was just asking for a mistake, especially since we use `ONE` elsewhere in `DaiHrd`.  I opted to drop it to make auditing easier.

Reply:
👍